docs/ref/models/querysets.txt

```diff
@@ -979,6 +979,16 @@ database.
     performance, since you have done a database query that you haven't used. So
     use this feature with caution!
 
+.. versionchanged:: 1.11
```

Comment:
Try to make these notes "self-contained" as described at https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features